external/runtime/runtime.depproj

```diff
@@ -83,7 +83,7 @@
       <_CoreCLRSymbolPackagesToDownload Include="@(ReferenceCopyLocalPaths->'%(NuGetPackageId)')" Condition="$([System.String]::Copy('%(Identity)').EndsWith('System.Private.CoreLib.dll'))">
         <!-- If there is an asset root URL set (for orchestrated build), then append symbols to that to grab the coreclr symbol location.  The orchestrated build uses a flat package layout. -->
         <Url Condition="'$(DotNetAssetRootUrl)'!=''">$(DotNetAssetRootUrl)symbols/%(NuGetPackageId).%(NuGetPackageVersion).symbols.nupkg</Url>
-        <Url Condition="'$(DotNetAssetRootUrl)'==''">https://dotnet.myget.org/F/dotnet-core/symbols/%(NuGetPackageId)/%(NuGetPackageVersion)/</Url>
+        <Url Condition="'$(DotNetAssetRootUrl)'==''">https://dotnetfeed.blob.core.windows.net/dotnet-core/assets/symbols/%(NuGetPackageId).%(NuGetPackageVersion).symbols.nupkg</Url>
```

Comment:
Could unify this with the other `Url` by assigning `https://dotnetfeed.blob.core.windows.net/dotnet-core/assets/` as the default asset root url.

Reply:
You're right. Will update. Thanks

Reply:
Fixed.